HCCGo/app/index.html

```diff
@@ -43,6 +43,7 @@
     <script src="lib/selectize/selectize.js"></script>
     <script src="lib/angular-toastr/angular-toastr.tpls.js"></script>
     <script src="lib/bootbox/bootbox.js"></script>
+    <script src="../../bower_components/ace-builds/src-noconflict/ace.js"></script>
```

Comment:
It seems awkward to address the ace.js like this.  Could you install it in the `lib` directory like the other javascript files?